### ISSUE TITLE
Add code coverage to our CI builds

### ISSUE THREAD

**Issue by jkonicki** [vmware-tanzu/tanzu-toolkit-for-visual-studio#13]
PT: 175462160

**Comment by atwoosnam** [comment]
As of 5/20 ~3pm we've got preliminary code coverage in our pipeline! 

The coverage rate at the moment is  around 85% -- this could be improved if we exclude some of our models (data contracts) from the coverage analysis (e.g. many models in the CfApiClient project have several untested fields, but that's because none of our business logic ever uses those fields -- they're relics of a "Copy JSON -> paste as C# Classes" operation & we've only kept them around because we feel like we might use them soon).

There is 1 remaining suite of tests that isn't represented in the coverage reports: `WpfViews.Tests`. This test project does not have Coverlet installed as a NuGet package so it doesn't get analyzed for coverage. Adding Coverlet to this project isn't trivial because it targets .NET Framework 4.7 -- we could add coverlet easily if it were a .NET Core project, but the conversion is complicated by a dependency on `PresentationFramework`. It should be possible to set up code coverage for our WpfViews in the future, but there are very few tests associated with the views at the moment anyway, so we're thinking of punting on this final test project for now.
